### ISSUE TITLE
Zoom 200% should still show all elements

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/vscode#52683]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.4.4

GitHubTags: #A11yMAS; #A11yTCS; #Win32; #DesktopApp #A11y_VSCodeEditor #MAS1.4.4 

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Open any file in editor.

**Actual:**
This application has no any defined max zoom like 200% or 250 like that.. and zoom always occur in the rightward direction due to which contents in the right part are not visible and loss of dta found in the bottom bar.

**Expected:**
Application should have some defined maximum zoom and when user zooms the application it should have scroll bar to view all the contents and zoom should be consistent and there should not be any loss of data.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}

Attachment for Reference: 
[A11y VSCode Editor Zoom.pptx](https://github.com/Microsoft/vscode/files/2127697/A11y.VSCode.Editor.Zoom.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@bpasero If I understand this correctly we should introduce a max zoom factor?
@AccessibilityTestingTeam-TCS what is the reason for that? Why should the zooming be bounded?

**Comment by isidorn** [comment]
@AccessibilityTestingTeam-TCS can you please answer my questoin

**Comment by AccessibilityTestingTeam-TCS** [comment]
No, it should not be bounded. User can zoom up to any extent according to his requirement but, all the contents and controls should be properly visible up to 200% zoom and it should resize without assistive technology(MAS Standard 1.4.4). Please check MAS Reference link in the repro steps.

**Comment by isidorn** [comment]
This is not how zoom works in chrome. There is nothing we can do about this, since going to 200% zoom some elements can always get lost -> renaming, upstream, backlog
